### ISSUE TITLE
Parent Selectors

### ISSUE THREAD

**Issue by tsedwards93** [martinsbalodis/web-scraper-chrome-extension#105]
Hello,

When I'm adding a new selector, I only get the option for "_root" in the "parent selectors" box. I need this to be the parent selector under _root.

Am I doing something wrong?

Thank you,
Tayler

**Comment by arunbanakar123** [comment]
Hi tsedwards93, 

Selector type - Text, Element Attribute, HTML and Grouped cannot be used as parent selector…, please ensure parent doesn’t having these selector types.

Iam i correct martinsbalodis....

**Comment by martinsbalodis** [comment]
Yes arunbanakar123 is right. Only Link selector, Element selector, Element click selector and Element scroll selector can have child selectors.
